makefile.shade

```diff
@@ -611,6 +618,18 @@ var CAN_BUILD_ONECORE = '${Directory.Exists(VS_ONECORE_ROOT) && Directory.GetFil
                          || (Path.GetFileName(f).StartsWith("System") && Path.GetFileName(f).EndsWith(nativeExtension))))
                 {
                     var targetFilePath = Path.Combine(target.TargetFolder, Path.GetFileName(file));
+                    if (file.EndsWith(nativeExtension))
+                    {
+                        foreach (var native in httpNativeAssemblies)
```

Comment:
Why not just blindly copy the files from httpNativeAssemblies instead of checking if the names match for each so/dylib?